src/main/java/wanion/avaritiaddons/block/chest/infinity/TileEntityInfinityChest.java

```diff
@@ -133,13 +138,13 @@ public ItemStack getStackInSlot(final int slot)
 	@Override
 	public boolean isItemValidForSlot(final int slot, final ItemStack itemStack)
 	{
-		return slot != 243;
+		return slot != 243 || findSlotFor(itemStack) != -1;
```

Comment:
is that number supposed to represent last slot index? What do you thin about moving it to a constant along with inventory size so it's a bit more clear?

What do you think about hiding `findSlotFor(itemStack) != -1` behind boolean method like `isPresentInTheChest(itemStack)` or `hasSlotWith(itemStack)`